crates/fuzzing/src/generators.rs

```diff
@@ -343,7 +343,6 @@ impl Config {
     pub fn set_spectest_compliant(&mut self) {
         let config = &mut self.module_config.config;
         config.memory64_enabled = false;
-        config.simd_enabled = false;
```

Comment:
FYI: this was removed as a cleanup because line 350 below has `config.simd_enabled = true;`